src/main/java/io/pravega/connectors/flink/FlinkPravegaReader.java

```diff
@@ -44,6 +52,26 @@
 
     private static final long serialVersionUID = 1L;
 
+    // ----- metrics field constants -----
+
+    private static final String PRAVEGA_READER_METRICS_GROUP = "PravegaReader";
+
+    private static final String READER_GROUP_METRICS_GROUP = "readerGroup";
```

Comment:
Seems that the various metrics here are all 'group-level', e.g. the unread bytes for the entire group.   However, Flink will report these metrics on a per-task basis.   I suppose that all the instances will be showing the same values (since they are all part of the same reader group).   I wonder whether we are missing a opportunity to report true instance-level metrics, e.g. how many segments are assigned to _this_ instance vs another.  Food for thought.